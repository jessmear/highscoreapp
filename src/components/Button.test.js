import React from 'react';
import { mount } from 'Enzyme';
import Button from './Button';

const buttonText = 'Test Button Text';
let wrapped = mount(<Button>{buttonText}</Button>);

describe('renders button', () => {
  it('should render the button component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the button children', () => {
    expect(wrapped.find('button').text()).toEqual(buttonText);
  });
});
