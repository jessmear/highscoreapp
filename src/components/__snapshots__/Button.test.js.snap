// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button should render the button component correctly 1`] = `
<Button>
  <button>
    Test Button Text
  </button>
</Button>
`;
